docs/appendices/release-notes/5.3.5.rst

```diff
@@ -47,4 +47,5 @@ See the :ref:`version_5.3.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Fixed ``IndexOutOfBoundsException`` caused by an ``is null`` filter on a
+  sub-column of an object in a ``WHERE`` clause.
```

Comment:
I would add the example here:
```
select * from t where a[1]['b'] is null; 
```

I think the description is not accurate, it's a sub-column of an object within an array, no?

Reply:
It also affects unknown object keys:

```
SELECT *
    FROM
      (SELECT 1 as id) AS foo
      JOIN
      testdrive.bar AS T
    ON
      foo.id = T.foo_id
    WHERE
      T.tags['unknown'] IS NOT NULL
```
basically for a few subscript expressions that are resolved to subscript functions instead of references.
I will rewrite with examples. Thanks!